wiki/Tournaments/OUKT/1/en.md

```diff
@@ -0,0 +1,298 @@
+
+### Semifinals
+
+Saturday, 4 January 2014
+
+| Player 1 | | | Player 2 | Notes |
+| --: | :-: | :-: | :-- | :-- |
+| **Doomsday** ![][flag_GB] | **4** | 1 | ![][flag_GB] Deathosaurus | |
+| ILikeMudkipz ![][flag_GB] | 0 | **1** | ![][flag_GB] **R a h a r u** | Forfeit |
+
+### Quaterfinals
+
+Saturday, 21 December 2013
+
+| Player 1 | | | Player 2 | Notes |
+| --: | :-: | :-: | :-- | :-- |
+| **Doomsday** ![][flag_GB] | **4** | 2 | ![][flag_GB] vidya | |
+| **Deathosaurus** ![][flag_GB] | **3** | 1 | ![][flag_GB] Starry- | Forfeit (physical problems) |
+| **ILikeMudkipz** ![][flag_GB] | **4** | 2 | ![][flag_GB] Viremon | |
+| **R a h a r u** ![][flag_GB] | **4** | 2 | ![][flag_GB] PortalLife | |
+
+### Round of 16
+
+Saturday, 14 December 2013
+
+| Player 1 | | | Player 2 | Notes |
+| --: | :-: | :-: | :-- | :-- |
+| **Doomsday** ![][flag_GB] | **3** | 0 | ![][flag_GB] Raiku | |
+| destructor966 ![][flag_GB] | 0 | **3** | ![][flag_GB] **vidya** | |
+| jesus1412 ![][flag_GB] | 0 | **1** | ![][flag_GB] **Deathosaurus** | Forfeit |
+| **Starry-** ![][flag_GB] | **3** | 1 | ![][flag_GB] \[Transcendence\] | |
+| **ILikeMudkipz** ![][flag_GB] | **1** | 0 | ![][flag_GB] Slvin | Forfeit |
+| Cozzzy ![][flag_GB] | 2 | **3** | ![][flag_GB] **Viremon** | |
+| **R a h a r u** ![][flag_GB] | **1** | 0 | ![][flag_GB] Nyaristoru | Forfeit |
+| Navi ![][flag_GB] | 2 | **3** | ![][flag_GB] **PortalLife** | |
+
+### Round of 32
+
+Saturday, 7 December 2013
+
+| Player 1 | | | Player 2 | Notes |
+| --: | :-: | :-: | :-- | :-- |
+| **Doomsday** ![][flag_GB] | **3** | 0 | ![][flag_GB] Zarfots | |
+| **Raiku** ![][flag_GB] | **3** | 1 | ![][flag_GB] Mahiro Yasaka | |
+| Charleyzard ![][flag_GB] | 0 | **1** | ![][flag_GB] **destructor966** | Forfeit |
+| **vidya** ![][flag_GB] | **3** | 0 | ![][flag_GB] Just miku | |
+| **jesus1412** ![][flag_GB] | **3** | 0 | ![][flag_GB] EzLyn | |
+| **Deathosaurus** ![][flag_GB] | **3** | 0 | ![][flag_GB] Aaruen | |
+| **Starry-** ![][flag_GB] | **1** | 0 | ![][flag_GB] 3lliot | Forfeit |
+| **\[Transcendence\]** ![][flag_GB] | **3** | 0 | ![][flag_GB] scouttt | |
+| **ILikeMudkipz** ![][flag_GB] | **1** | 0 | ![][flag_GB] ItzDom | Forfeit |
+| **Slvin** ![][flag_GB] | **3** | 2 | ![][flag_GB] Snookie | |
+| **Cozzzy** ![][flag_GB] | **3** | 0 | ![][flag_GB] Yoishi | |
+| bubby963 ![][flag_GB] | 0 | **1** | ![][flag_GB] **Viremon** | Forfeit |
+| **R a h a r u** ![][flag_GB] | **3** | 0 | ![][flag_GB] Tekklorn | |
+| Kardet ![][flag_GB] | 2 | **3** | ![][flag_GB] **Nyaristoru** | |
+| **Navi** ![][flag_GB] | **3** | 0 | ![][flag_GB] Riari | |
+| **PortalLife** ![][flag_GB] | **3** | 1 | ![][flag_GB] Lusory | |
+
+## Ruleset
+
+### Tournament Rules
+
+- This tournament is a single elimination 1v1 tournament. Lose once, you are out. Brackets have been determined by seeding via TP. (There is one exception here, as someone took the place of a higher seed player.)
```

Comment:
TP would have been osu!tp since it would have been the preferred system of judging a players skill over ppv1. It doesn't really feel appropriate to link ppv2 since the system isn't 1:1 even if they were both nearly the same when osu!tp moved over to ppv2

Reply:
```suggestion
- This tournament is a single elimination 1v1 tournament. Lose once, you are out. Brackets have been determined by seeding via [osu!TP](https://web.archive.org/web/20131113173042/http://osutp.net/info "Internet Archive - osu!tp). (There is one exception here, as someone took the place of a higher seed player.)
```

The actual website is dead so this is probably the best explanation we are going to get.

Reply:
works for me!